bullyBot.py

```diff
@@ -0,0 +1,47 @@
+#bullyBot.py
+import os
+import discord
+from discord.ext import commands
+from dotenv import load_dotenv
+
+load_dotenv()
+TOKEN = os.getenv('DISCORD_TOKEN')
+GUILD = os.getenv('DISCORD_GUILD')
+
+bot = commands.Bot(command_prefix='!')
+
+@bot.event
+async def on_ready():
+    for guild in bot.guilds:
+        if guild.name == GUILD:
+            break
+
+    print(
+        f'{bot.user.name} is connected to the following guild:\n'
+        f'{guild.name}(id: {guild.id})'
+    )
+
+@bot.command(name='bully', help='Creates a new channel for bullying a new member')
+async def on_message(ctx, victim):
+    
+
+
+    victim=victim.lower().strip()
+    channel_name=f'making-fun-of-{victim}'
+    guild = ctx.guild
+    existing_channel = discord.utils.get(guild.channels, name=channel_name)
+    bullying_category = discord.utils.get(guild.categories, name='bullying')
+
+    if not existing_channel:
+        response = f'Good choice, {victim} is due for a bullying'
+        #print(f'Creating a new channel for the purpose of bullying {victim}.')
```

Comment:
General rule: unprofessional to commit commented out code.  Decided whether or not we want this debug message.

Reply:
Thanks for the feedback. I did not know that rule.

Reply:
+1